test/org/sagebionetworks/bridge/validators/SurveyValidatorTest.java

```diff
@@ -380,6 +383,84 @@ public void willValidateRuleReferencesToNonQuestions() {
     }
     
     @Test
+    public void willVerifyRuleDoesntHaveSkipToAndEndSurvey() throws Exception {
```

Comment:
Is it worth also adding a test to make sure a valid rule with getEndSurvey() passes validation?